usaspending_api/settings.py

```diff
@@ -107,11 +107,19 @@
 
 DATABASES = {'default': dj_database_url.config(conn_max_age=10)}
 
+# read replica env vars... if not set, default DATABASE_URL will get used
+# if only one set, this will error out (single DB should use DATABASE_URL)
+if os.environ.get('DB_SOURCE') or os.environ.get('DB_R1'):
```

Comment:
@willkjackson I know this PR has already been merged, but adding a note that we may want to explicitly exclude these new databases from the test suite. I had previously run into an issue where pytest was setting up extra test databases everywhere because I had the BROKER environment variable set.

Pinging @afrasier and @catherinedevlin for their thoughts. The place we ended up excluding the broker db from the test suite is: https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/conftest.py#L9